docs/deployment/add-semgrep-to-ci.md

```diff
@@ -0,0 +1,132 @@
+---
+slug: add-semgrep-to-ci
+title: Add Semgrep to CI/CD
+hide_title: true
+description: "Set up your CI pipeline with Semgrep Cloud Platform for centralized rule and findings management."
+tags:
+  - Deployment
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+import PlatformAddRepo from "/src/components/procedure/_platform-add-repo.md"
+import PlatformDetectGhRepos from "/src/components/procedure/_platform-detect-ghrepos.md"
+import NextStepsComments from "/src/components/concept/_next-steps-comments.mdx"
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Add Semgrep to CI
+
+:::note Your deployment journey
+- You have gained the necessary [resource access and permissions](/deployment/checklist) required for deployment.
+- You have [created a Semgrep account and organization](/deployment/create-account-and-orgs). 
+- For GitHub and GitLab users: You have [connected your source code manager](/deployment/connect-scm).
+- Optionally, you have [set up SSO](/deployment/sso).
+:::
+
+Semgrep is integrated into CI environments by creating a **job** (also known as an **action** for some CI providers) that is run by the CI provider. After a scan, findings are sent to Semgrep Cloud Platform for triage and remediation. 
+
+This guide walks you through creating a Semgrep job in the following **in-app CI providers**:
```

Comment:
Good callout. Yeah I struggled with a descriptor.

For **Supported** - we... "technically" support any CI provider, so I couldn't use it.

My alternative was "CI providers with Semgrep-provided configurations" but axed it because it was a mouthful. Terms like **guided** or **assisted** set up seems to be on the right track.

Reply:
I like "guided"!